### PR TITLE
docs(*): fill out roles index and overviews for each

### PR DESCRIPTION
**Pull request by vdice**
* Adds documentation around roles in Brigade
* ~~Also snuck in a placeholder for a [Metrics](https://github.com/brigadecore/brigade-metrics) doc 😄~~

TODO
  - [x] Get linking working right 😂 

Closes https://github.com/brigadecore/brigade/issues/1509

### CODE REVIEW

File: docs/config.toml [hunk 1]
```diff
@@ -158,6 +158,11 @@ relativeURLs = "true"
     weight = 70
     parent = "scripting"
     [[menu.index]]
+    name = "Metrics"
+    url = "/topics/metrics/"
+    weight = 74
+    parent = "topics"
+    [[menu.index]]
```
Comment (krancour) [inline docs/config.toml:165]: Brigade Metrics, for now, is a separate project and basically a bolt-on. Does that influence whether we should or shouldn't cover it here in these docs? It's not strongly held, but I think my opinion is that it's only worthy of a brief mention in a section on other things in the extended Brigade 2 ecosystem.

File: docs/content/topics/administrators/_index.md [hunk 2]
```diff
@@ -3,9 +3,24 @@ title: Administrators
...
 ---
 
 # Brigade Administrators
 
-TODO
\ No newline at end of file
+Users who are in charge of authentication and authorization concerns within the
+Brigade system are known as Administrators.
+
+They'll select a supported third-party authentication strategy to plug into
```
Comment (krancour) [inline docs/content/topics/administrators/_index.md:15]: They strategy would have had to be selected by the operator at deploy time.

File: docs/config.toml (hunk 1 above)
Comment (vdice) [reply docs/config.toml]: Fair point; removed for now.

File: docs/content/topics/administrators/authentication.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
 ---
 title: Authentication
 description: Authentication configuration for Brigade
+aliases:
+  - /authentication
+  - /topics/authentication.md
+  - /topics/administrators/authentication.md
```
Comment (krancour) [inline docs/content/topics/administrators/authentication.md:7]: Isn't this an operator concern?

File: docs/content/topics/administrators/_index.md [hunk 4]
```diff
@@ -3,9 +3,26 @@ title: Administrators
...
+  - /topics/administrators
   - /topics/administrators/index.md
 ---
 
 # Brigade Administrators
 
-TODO
\ No newline at end of file
+Users who are in charge of authentication and authorization concerns within the
+Brigade system are known as Administrators.
```
Comment (krancour) [inline docs/content/topics/administrators/_index.md:13]: I'm worried this definition isn't general enough.

What if we present admins as users who are responsible for _managing_ an instance of Brigade?

Maybe also worth noting that this role may often, but not always, be fulfilled by the same person/people fulfilling the operator role?

Comment (krancour) [reply docs/content/topics/administrators/_index.md:13]: The concern about this not being a general enough description repeats in a few other spots.

File: docs/content/topics/administrators/authentication.md (hunk 3 above)
Comment (vdice) [reply docs/content/topics/administrators/authentication.md:7]: This is one that definitely straddles both roles.  I would think admins would want a choice over which third-party auth system is deployed (by an operator), as it relates to users which they'll set up authz around... but I'm down to move it if we feel strongly.

File: docs/content/topics/administrators/_index.md (hunk 4 above)
Comment (vdice) [reply docs/content/topics/administrators/_index.md:13]: Indeed, I added the following text to the roles doc:

```
Naturally, there may be overlap between these roles, and for development
setups, one or two users might cover all of them.  However, they also serve
nicely as categories of context for documentation, so without further ado,
let's dive in.
```

But I can certainly add role-specific notes to each index, such as the one you've pointed out here, where the admin role might naturally be filled by the operator.

File: docs/content/topics/administrators/authentication.md (hunk 3 above)
Comment (krancour) [reply docs/content/topics/administrators/authentication.md:7]: Interesting point. I can imagine admins wanting some say in the matter. Let's roll with it as is for now and we can change it later if necessary.

**Review by krancour — APPROVED** [review]

**Pull request merged**